cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -751,6 +751,9 @@ <h2 class="h3 u-mb15">
                         {% elif card.transfer_apr_median is not none %}
                             Median balance transfer APR
                         {% endif %}
+                        for
+                        {{ '%g' | format(card.median_length_of_balance_transfer_apr | round) }}
```

Comment:
Same question as above about `round` - I don't see any examples of non-integer values for this in either dataset, fwiw.